src/components/MovieDetails/index.tsx

```diff
@@ -245,8 +254,8 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
         onClose={() => setShowManager(false)}
         subText={data.title}
       >
-        {((data?.mediaInfo?.downloadStatus ?? []).length > 0 ||
-          (data?.mediaInfo?.downloadStatus4k ?? []).length > 0) && (
+        {(!!(data?.mediaInfo?.downloadStatus ?? []).length ||
```

Comment:
Why exactly are we refactoring all of these conditionals? 🤔

Reply:
Ah, I was adopting the new style TheCatLady was using in her PR #1778. I can definitely leave out this change if preferred. 😊

Reply:
Oops, removed from this PR